src/antsibull_core/venv.py

```diff
@@ -153,8 +154,49 @@ class FakeVenvRunner:
         * :python:mod:`venv`
     """
 
-    log_run = staticmethod(subprocess_util.log_run)
-    async_log_run = staticmethod(subprocess_util.async_log_run)
+    @staticmethod
+    async def async_log_run(
+        args: MutableSequence[StrPath],
+        logger: TwiggyLogger | StdLogger | None = None,
+        stdout_loglevel: str | None = None,
+        stderr_loglevel: str | None = 'debug',
+        check: bool = True,
+        *,
+        errors: str = 'strict',
+        **kwargs,
+    ) -> subprocess.CompletedProcess[str]:
+        """
+        This method asynchronously runs a command in a subprocess and logs its
+        output.
+        It works the same as `antsibull_core.subprocess_util.async_log_run`,
+        but 'python' will be replaced by `sys.executable`.
+        """
+        if args and args[0] == 'python':
+            args[0] = sys.executable
```

Comment:
Why not create a copy instead of modifying the original sequence?
```suggestion
            args = args[:]
            args[0] = sys.executable
```

Reply:
I changed all of the code in the module that mutated `args`.